.gitignore

```diff
@@ -136,7 +136,9 @@ key.pem
 
 # test artifacts
 reports/
-/static/
+/static/*
+# allow for docs to be downloaded
+!/static/pdf/
```

Comment:
We probably still want to git ignore this in the root, it’s the sub directory where this gets collected from